ui/jobs/jobs.js

```diff
@@ -3124,6 +3226,48 @@ class Bars {
           this.buffTracker.onUseAbility(m.groups.id, m.groups);
       }
     }
+    // For extremely complex BRD
+    if (this.job != 'BRD') return;
+    if (!this.dotTarget)
+      this.dotTarget = [];
+    const brdDoTs = Object.freeze([
+      EffectId.Stormbite,
+      EffectId.Windbite,
+      EffectId.CausticBite,
+      EffectId.VenomousBite,
+    ]);
+    if (type === '26') {
+      let m = log.match(kMobGainsEffectFromYouRegex);
+      if (m) {
+        if (Object.values(brdDoTs).includes(m.groups.effectId.toUpperCase()))
```

Comment:
If you're going to uppercase the id twice, could you save it in a local `effectId` variable?